### ISSUE TITLE
Launch Page: Add a link to the documentation

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-launch-dapp#507]

**Comment by BartuCulha** [comment]
Is Jaap goig to provide a mock for this one? @renc-curvelabs

**Comment by renc-curvelabs** [comment]
No. We will just add the sentences mentioned above to the end of the explanations under each section:

So Seed should say:

Raise a seed from your community and early supporters publicly or via a whitelist. Visit here to learn more about Seed Launch." 
"here" should be hyperlinked and leading to https://launch.prime.xyz/documentation/seed-launch

LBP should say: 

"The liquid launch powered by Balancer Protocol allows for fair price discovery and creates initial liquidity for your token. Visit here to learn more about Liquid Launch." 
"here" should be hyperlinked and leading to https://launch.prime.xyz/documentation/liquid-launch

**Comment by BartuCulha** [comment]
ok moving card to triage

**Comment by dkent600** [comment]
@renc-curvelabs Can you please specify whether or not these links should open in a new tab?

**Comment by renc-curvelabs** [comment]
They should open in a new tab sir - thanks for asking

**Comment by renc-curvelabs** [comment]
Cannot see the changes here:
https://primelaunchtest.primedao.io/register

**Comment by dkent600** [comment]
@wantedsystem This is still not complete
